### DISCUSSION TITLE
Is there any way of addid a dropdown picker for year and month in calendars?

### DISCUSSION THREAD

**Discussion by samchouse** [adobe/react-spectrum#6444] [Q&A] [answered]
Using react-day-picker, I'm able to add a month and year selector to be able to quickly pick dates. Is there any way of doing this with react-aria? At the moment, I'm unable to find any methods that can be called to set the year/month in my state object.

![image](https://github.com/adobe/react-spectrum/assets/46873232/13e36328-3f42-4201-9eb6-93d516d67cb5)

**Comment by devongovett** [comment, marked as answer]
You can do this by controlling the focused date via the `focusedValue` prop: https://react-spectrum.adobe.com/react-aria/Calendar.html#controlling-the-focused-date
